node/src/server-modules/GlideJson.ts

```diff
@@ -1157,3 +1158,778 @@ export class GlideJson {
+     */
+    static set(
+        transaction: Transaction | ClusterTransaction,
+        key: GlideString,
+        path: GlideString,
+        value: GlideString,
+        options?: { conditionalChange: ConditionalChange },
+    ): Transaction | ClusterTransaction {
+        const args: GlideString[] = ["JSON.SET", key, path, value];
+
+        if (options?.conditionalChange !== undefined) {
+            args.push(options.conditionalChange);
+        }
+
+        return transaction.customCommand(args);
+    }
+
+    /**
+     * Retrieves the JSON value at the specified `paths` stored at `key`.
+     *
+     * @param transaction - A transaction to add commands to.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) Options for formatting the byte representation of the JSON data. See {@link JsonGetOptions}.
+     *
+     * Command Response -
+     *   - If one path is given:
+     *     - For JSONPath (path starts with `$`):
+     *       - Returns a stringified JSON list of bytes replies for every possible path,
+     *         or a byte string representation of an empty array, if path doesn't exist.
+     *         If `key` doesn't exist, returns `null`.
+     *     - For legacy path (path doesn't start with `$`):
+     *         Returns a byte string representation of the value in `path`.
+     *         If `path` doesn't exist, an error is raised.
+     *         If `key` doesn't exist, returns `null`.
+     *  - If multiple paths are given:
+     *         Returns a stringified JSON object in bytes, in which each path is a key, and it's corresponding value, is the value as if the path was executed in the command as a single path.
+     * In case of multiple paths, and `paths` are a mix of both JSONPath and legacy path, the command behaves as if all are JSONPath paths.
+     */
+    static get(
+        transaction: Transaction | ClusterTransaction,
+        key: GlideString,
+        options?: JsonGetOptions,
+    ): Transaction | ClusterTransaction {
+        const args = ["JSON.GET", key];
+
+        if (options) {
+            const optionArgs = _jsonGetOptionsToArgs(options);
+            args.push(...optionArgs);
+        }
+
+        return transaction.customCommand(args);
+    }
+
+    /**
+     * Retrieves the JSON values at the specified `path` stored at multiple `keys`.
+     *
+     * @remarks When in cluster mode, if keys in `keyValueMap` map to different hash slots, the command
+     * will be split across these slots and executed separately for each. This means the command
+     * is atomic only at the slot level. If one or more slot-specific requests fail, the entire
+     * call will return the first encountered error, even though some requests may have succeeded
+     * while others did not. If this behavior impacts your application logic, consider splitting
+     * the request into sub-requests per slot to ensure atomicity.
```

Comment:
```
@remarks When in cluster mode, all keys in the transaction must be mapped to the same slot.
```

Reply:
This is the remark on BaseClient mget as well. Should that also be updated?

Reply:
No
transaction goes to only 1 node always, but a command executed outside of transaction could be routed to multiple nodes.
Good finding - should be updated for mget in transaction for all clients

Reply:
Agreed.  I have made a comment on 3 PRs.

Reply:
Same comment could be applied to all multi-key commands.

Reply:
I mean JSON.MGET
I remember we were not adding same slot notice in transaction, but I don't remember the reason...